app/controllers/content_items_controller.rb

```diff
@@ -23,8 +23,9 @@ def load_content_item
   def present(content_item)
     case content_item['format']
     when 'case_study' then CaseStudyPresenter.new(content_item)
-    when 'take_part' then TakePartPresenter.new(content_item)
     when 'statistics_announcement' then StatisticsAnnouncementPresenter.new(content_item)
+    when 'take_part' then TakePartPresenter.new(content_item)
+    when 'topical_event_about_page' then TopicalEventAboutPagePresenter.new(content_item)
```

Comment:
It feels to me like there's enough duplication here to use convention over configuration (ie to derive the presenter class name from the format name):

```ruby
def present(content_item)
  presenter_name = content_item['format'].classify + 'Presenter'
  presenter_class = Object.const_get(presenter_name)
  presenter_class.new(content_item)
rescue NameError
  raise "No support for format \"#{content_item['format']}\""
end
```